pkg/sqlite/graph_loader.go

```diff
@@ -0,0 +1,159 @@
+package sqlite
```

Comment:
this is already merged into master with a new file name fyi: https://github.com/operator-framework/operator-registry/blob/master/pkg/sqlite/graphloader.go#L11

Reply:
It's because you merged their commits into your repo and are pulling them along, but i squashed their commits and then renamed this file to get rid of the underscore in the name. you need to remove their commits and rebase.